image_tools/src/showimage.cpp

```diff
@@ -206,8 +209,8 @@ class ShowImage : public rclcpp::Node
 
       cv::Mat cvframe = frame;
 
-      // Show the image in a window called "showimage".
-      cv::imshow("showimage", cvframe);
+      // Show the image in a window
+      cv::imshow(window_name_, cvframe);
```

Comment:
IMO, it would be more elegant to use the topic name as the window name. This has the effect the number of windows will equal the number of image topics being subscribed to by all `showimage` nodes. It also avoids a user forgetting to set the window name parameter.

I think this should work (I haven't tested it):

```suggestion
      cv::imshow(sub_->get_topic_name(), cvframe);
```